src/com/projectkorra/projectkorra/ability/ElementalAbility.java

```diff
@@ -132,15 +157,20 @@ public static boolean isMetal(final Block block) {
 	}
 
 	public static boolean isMetal(final Material material) {
-		return getConfig().getStringList("Properties.Earth.MetalBlocks").contains(material.toString());
+		List<String> metalBlocks = getConfig().getStringList("Properties.Earth.MetalBlocks");
+		for (String tag : getConfig().getStringList("Properties.Earth.MetalBlocks")) {
+			if (tag.startsWith("#")) {
+				metalBlocks.addAll(GeneralMethods.tagToMaterials(tag));
+			}
+		}
+		return metalBlocks.contains(material.toString());
 	}
 
 	public static boolean isMetalBlock(final Block block) {
-		if (block.getType() == Material.GOLD_BLOCK || block.getType() == Material.IRON_BLOCK || block.getType() == Material.IRON_ORE || block.getType() == Material.GOLD_ORE || block.getType() == Material.QUARTZ_BLOCK || block.getType() == Material.NETHER_QUARTZ_ORE) {
-			return true;
-		}
-
-		return false;
+//		if (block.getType() == Material.GOLD_BLOCK || block.getType() == Material.IRON_BLOCK || block.getType() == Material.IRON_ORE || block.getType() == Material.GOLD_ORE || block.getType() == Material.QUARTZ_BLOCK || block.getType() == Material.NETHER_QUARTZ_ORE) {
+//			return true;
+//		}
+		return isMetal(block.getType());
```

Comment:
I left the method in case it's being used elsewhere by anyone else. I didn't write the original redundant method, sorry!

Reply:
Oh no, I'm aware, all I'm asking you is that if you could

Replace line 173 with the following `return isMetal(block);`, so that way it doesn't circumvent the check we put to see whether its null when isMetal is used on a Block type rather than a Material type

Reply:
Allowing a caller to have options isn't redundant and can often shorten their code. @Varhagna request is that, for this telescopic call, utilizing the null safe variant for `block` would be preferred.